app/code/core/Mage/Catalog/Block/Widget/Link.php

```diff
@@ -79,7 +78,7 @@ public function getHref()
                 $categoryId = $idPath[1];
                 if ($categoryId) {
                     /** @var Mage_Catalog_Helper_Category $helper */
-                    $helper = $this->_getFactory()->getHelper('catalog/category');
+                    $helper = $this->_getFactory()->getHelper('catalog/product');
```

Comment:
was this correct to change the model from category to product?

<img width="643" alt="Schermata 2022-08-19 alle 00 01 47" src="https://user-images.githubusercontent.com/909743/185509722-3d4d26f5-4059-4935-8e3e-d8c4ac54830c.png">

Reply:
No, this was not correct. Mistake when reverting ... sorry.